src/internal/transactionenv/env.go

```diff
@@ -84,13 +73,6 @@ type TransactionContext struct {
 	txnEnv         *TransactionEnv
 }
 
-// Auth returns a reference to the Auth API Server so that transactionally-
-// supported methods can be called across the API boundary without using RPCs
-// (which will not maintain transactional guarantees)
-func (t *TransactionContext) Auth() AuthTransactionServer {
-	return t.txnEnv.authServer
-}
-
```

Comment:
Moved this to serviceenv since the interface contains transaction + non-transaction methods.